mobile/Ihc/util/Sync.js

```diff
@@ -11,3 +11,12 @@ export function downstreamSyncWithServer() {
       return {failedPatientKeys: failedPatientKeys};
     });
 }
+export function downloadMedications() {
+  const lastSynced = localData.lastSynced();
```

Comment:
Hmm so this lastSynced is currently for the patients only. In HandleDownloadedPatients if the download is successful, then we update the lastSynced timestamp. However I don't think you should just update the lastSynced timestamp in your HandleDownloadedMedications function because then when we download the medications, then the timestamp will indicate that the patients were updated too, but they weren't. 
Thus there are two options:
1. split into two settings fields (patientsLastSynced/medicationsLastSynced)
2. Download the medications and the patients at the same time, and update the single lastSynced timestamp after it is all complete

I have a preference for the first option because it will be more modular and we won't have to mix Medications with Patients because they aren't necessarily tied together. I can merge this before that gets done, but I wanna hear your opinion first